test/e2e/cli-plugin-install.e2e.test.js

```diff
@@ -0,0 +1,84 @@
+const fs = require('fs');
+const path = require('path');
+const util = require('util');
+
+const assert = require('chai').assert;
+const cpr = require('cpr');
+const rimraf = require('rimraf');
+const tmp = require('tmp');
+const yaml = require('js-yaml');
+
+require('util.promisify/shim')();
+
+const { runCLICommand } = require('../common/cli.helper');
+
+const gatewayDirectory = path.join(
+  __dirname,
+  '..',
+  'fixtures',
+  'plugin-installer'
+);
+
+const PACKAGE_NAME = 'express-gateway-plugin-test';
+
+const pluginDirectory = path.join(
+  __dirname,
+  '..',
+  'fixtures',
+  PACKAGE_NAME
+);
+
+let tempPath = null;
+let configPath = null;
+
+const config = {
+  systemConfigPath: null,
+  gatewayConfigPath: null
+};
+
+describe('E2E: eg plugins install', () => {
```

Comment:
Testing frameworks are usually relying on `bind` a lot to inject a `this` object you can use down in the chain. If you're using arrow functions, that won't work. I usually stick with `function() {}` here.

Reply:
I know, but I'm not using `this` in this particular test.  `function` is just sooo many more characters.  :wink:

Reply:
Keyboard strokes are expansive these days 😱